src/traces/barpolar/attributes.js

```diff
@@ -69,6 +71,9 @@ module.exports = {
     marker: barAttrs.marker,
 
     hoverinfo: scatterPolarAttrs.hoverinfo,
+    hovertemplate: hovertemplateAttrs({}, {
```

Comment:
```
hovertemplate: hovertemplateAttrs(),
```

is probably good enough for all the trace types patched in this PR. That extra `keys` list is meant for traces with event data keys that do not match coordinates (e.g. histogram traces). No need to add new `constants.js` files as well.